esileapclient/osc/v1/lease_contract.py

```diff
@@ -40,10 +40,10 @@ def get_parser(self, prog_name):
             required=False,
             help="Name of the contract being created. ")
         parser.add_argument(
-            '--offer-uuid',
-            dest='offer_uuid',
+            '--offer-uuid-or-name',
+            dest='offer_uuid_or_name',
```

Comment:
I think I would just call this "offer", with the help text explaining what would be expected.

Reply:
fixed